model_api/python/openvino/model_api/adapters/onnx_adapter.md

```diff
@@ -0,0 +1,56 @@
+# ONNX Runtime Adapter
+
+The `ONNXRuntimeAdapter` implements `InferenceAdapter` interface. The `ONNXRuntimeAdapter` allows Model API to leverage ONNX Runtime for inference.
+
+## Prerequisites
+
+`ONNXRuntimeAdapter` enables inference via ONNX Runtime, and we need to install it first:
+```sh
+pip install onnx onnxruntime
+```
+
+### ONNX metadata
+
+ModelAPI uses IR RTInfo to store metadata (wrapper-specific parameters, preprocessing parameters, labels list, etc.).
+For details see the implementation of `Model._load_config()` method.
+To embed that metadata into ONNX file, one can use ONNX properties: `metadata_props.add()` and use
+ModelAPI-specific parameters as metadata keys with exactly the same names as in RTInfo, but split by spaces: `"model_info model_type"` and so on.
+
+## Limitations
+
+- Although `ONNXRuntimeAdapter` doesn't use OpenVINO directly, OV should be installed, because Model API depends on it at the low level.
```

Comment:
OpenVINO is in the mandatory dependencies. I don't think you should mention it here.

Reply:
> I would note instead that there is only Python version of ORT adapter.

Done.

Regarding the deps notice, it was targeted to onnx fans among Geti users: from their side it's not that clear why do we need to install OV if the inference is conducted via ONNXRT.